Package.swift

```diff
@@ -22,11 +22,12 @@ func resolveTargets() -> [Target] {
         .target(name: "Purchases",
                 dependencies: [],
                 path: ".",
-                sources: ["Purchases", "Purchases/Public"],
```

Comment:
Ouch. That's going to happen again. Do you think there's a way to say "Everything under the Purchases folder"

Reply:
well, sources is actually recursive, so I think we don't actually need this line. 
for some reason, though, it works if you remove caching from this line, but not if you remove public. 
the one that there's pretty much no way around is the header search path... but maybe that one can take in a regular expression, I haven't checked yet

Reply:
I'm doing another pass on this for the universal purchases thing, since xcode 11.4 complains in any case. but for now, this should work for anyone using xcode <= 11.3 and unblock them.